### ISSUE TITLE
64bit?

### ISSUE THREAD

**Issue by thatvirtualboy** [SpottyMatt/raspian-3b-stepmania-arcade#14]
Hello! Has anyone tested this with the 64bit Raspberry Pi OS on the latest 8GB model?

**Comment by AlexGellert-zz** [comment]
Yes and so far my only issue is my HDMI is not receiving stepmania audio. Trying to debug so now.

**Comment by thatvirtualboy** [comment]
Sweet! Would love to know more about your experience. For now I’m running the latest 32bit OS and just got SM up and running tonight on a new 8gb model using a Ugreen USB sound card.  

Huge thanks to Matt for this script and the guide!

**Comment by AlexGellert-zz** [comment]
So it pretty much sets itself up, just git clone https://github.com/SpottyMatt/raspbian-stepmania-arcade and type make in the source folder. I'm not sure if it was the overclocking or this https://github.com/SpottyMatt/raspbian-stepmania-arcade#raspberry-pi-4b that really increased performance

**Comment by SpottyMatt** [comment]
@AlexGellert , in my experience the biggest increase in performance comes from dropping the StepMania resolution even just one notch below your monitor's native. It usually still looks okay, but you gain _so much_ performance.

Spec-wise, the RPi4b comes out of the box at just around (or over, depending) the maximum possible overclock of the 3b, you might think that's the second biggest performance boost you can do and it is, but not because of the CPU clock. In my experience, the limiting factor for StepMania on 3b was (graphics) _memory_ - and the RPi4 has so much more memory available that simply having a 4B solves most of the performance issues that we had to fight tooth-and-nail for on the 3b.

**Comment by andrew-lewin** [comment]
I just attempted this on one of the Raspberry Pi OS 64-bit beta versions. When it was compiling the stepmania code, it ran into an issue where it couldn't find `sys/io.h`. After some googling, it turns out that arm64 indeed does not include `sys/io.h`, so I'm looking around for what I can replace it with.
